pkg/apiserver/apiserver.go

```diff
@@ -309,6 +309,10 @@ func installHandlers(c *ExtraConfig, s *genericapiserver.GenericAPIServer) {
 		})
 	}
 
+	if features.DefaultFeatureGate.Enabled(features.Egress) || features.DefaultFeatureGate.Enabled(features.ServiceExternalIP) {
+		s.Handler.NonGoRestfulMux.HandleFunc("/validate/externalippool", webhook.HandlerForValidateFunc(c.externalIPPoolController.ValidateExternalIPPool))
```

Comment:
The handler will be installed twice if you don't remove the below one.

Reply:
deleted the one below. thanks.